scripts/asciidoc2html.py

```diff
@@ -19,6 +19,7 @@
 # along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
 
 """Generate the html documentation based on the asciidoc files."""
+from typing import List, Tuple
```

Comment:
nitpick: Please move this down to the other imports, as it's a Python stdlib import.

Reply:
You mean skip the blank line between this and the other stdlib imports?

Reply:
Yeah (and keep the blank line between the docstring above and the imports)